src/map/status.cpp

```diff
@@ -10126,7 +10127,9 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 			break;
 
 		case SC_STONEWAIT:
-			val3 -= tick; // Petrify time - Incubation time
+			val4 = val3;
+			val3 = tick - val3; // Petrify time - Incubation time
```

Comment:
```suggestion
			val3 = max(1, tick - val3); // Petrify time - Incubation time
```
Prevent any possibility of a negative value.